zipkin-collector/kafka/README.md

```diff
@@ -44,3 +44,17 @@ for (int i = 0; i < count; i++) {
 ### Legacy encoding
 Older versions of zipkin accepted a single span per message, as opposed
 to a list per message. This practice is deprecated, but still supported.
+
+## Kafka topic settings
+By default, this collector creates one instance of KafkaConsumer to poll for messages. Given that,
+our suggestion is to create the `zipkin` topic with one partition initially. Observe the KafkaConsumer
```

Comment:
Not sure if suggesting 1 topic partition is a good default, as it will involve manually changing number of partitions and scaling consumers.
6 could be a good starting point for production workloads as it gets evenly assigned to 1, 2, or 3 instances.